src/IECoreHoudini/GEO_CortexPrimitive.cpp

```diff
@@ -455,18 +500,25 @@ void GEO_CortexPrimitive::copyPrimitive( const GEO_Primitive *src )
 	}
 	
 	const GEO_CortexPrimitive *orig = (const GEO_CortexPrimitive *)src;
-	const GA_IndexMap &srcPoints = orig->getParent()->getPointMap();
-	
+
 	/// \todo: should we make a shallow or a deep copy?
 	m_object = orig->m_object;
+
+	GEO_Primitive::copyPrimitive(src);
```

Comment:
Why do we call the base class implementation after setting m_object? Did it exist in H15 and if so, how does this change behaviour?